.gitignore

```diff
@@ -8,6 +8,8 @@ dependency-reduced-pom.xml
 /wpilibj/src/athena/cpp/nivision/*.java
 /wpilibj/src/athena/cpp/nivision/nivision_funcs.txt
 /wpilibj/src/athena/cpp/nivision/imaqdx_funcs.txt
+wpilibj/src/shared/java/edu/wpi/first/wpilibj/util/WPILibVersion.java
+wpilibc/shared/src/WPILibVersion.cpp
```

Comment:
Without a leading slash, git will ignore in every path, not just at the top level (e.g. if you ignore `/foo`, only the top-level `/foo` will get ignored, if you ignore `foo`, `/foo`, `/bar/foo`, `/baz/foo` etc will all be ignored.